packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -2,17 +2,44 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { Text } from '@deriv/components';
 import { BinaryLink } from '../../Routes';
+import { routes } from '@deriv/shared';
+import { useHistory } from 'react-router';
+
+const MenuItems = ({ item, hide_menu_item, toggleReadyToDepositModal, has_any_real_account, is_virtual }) => {
```

Comment:
@thisyahlen-deriv This component has been refactored in the `develop` branch, Not sure how we gonna handle the conflict later while merging `master` and `develop` 👀 Maybe it would be better to create PR against `develop` 🤔

Reply:
Squad production QA cards are supposed to go to master 😿